src/services/giant_api.rs

```diff
@@ -14,8 +14,16 @@ use crate::{
     },
 };
 
+use clap::ValueEnum;
+
 use urlencoding::encode;
 
+#[derive(ValueEnum, Clone)]
+pub enum ListBlobsFilter {
+    All,
+    InMultiple,
```

Comment:
Maybe a `///` doc comment would help elucidate what this means to a future developer?

Reply:
good call, have added